influxdb/resource_database_test.go

```diff
@@ -183,34 +183,30 @@ resource "influxdb_database" "test" {
 
 var testAccDatabaseWithRPSConfig = `
 resource "influxdb_database" "rptest" {
-  name = "terraform-rp-test"
-  retention_policies = [
-    {
-      name = "1day",
-      duration = "1d"
-      default = "true"
-    },
-    {
-      name = "52weeks",
-      duration = "52w",
-    }
-  ]
+	name = "terraform-rp-test"
```

Comment:
Obviously not a blocker, but these inline HCL fragments are usually formatted with spaces, like `terraform fmt` does. It seems that here they jumped to tabs, presumably because of Go fmt.